src/redux/books/books.js

```diff
@@ -47,3 +40,31 @@ export const removeBook = (book) => ({
   type: REMOVE,
   book,
 });
+
+// API requests
+export const getBooks = createAsyncThunk(GET, async () => {
+  const response = await fetch(url);
+  const data = await response.json();
+  return data;
+});
+
+export const createBook = createAsyncThunk(ADD, async (book) => {
+  const response = await fetch(url, {
+    method: 'POST',
+    body: JSON.stringify(book),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  console.log(response);
```

Comment:
- [ ] @paulinagonzalezc great work on your rcreateBook function but you forgot to remove the console.log, its not good practice to have it in our codebase

Reply:
Thank you @danmainah, I will remove it.